datasets/gov/kauno_m_sav/kasimai.csv

```diff
@@ -0,0 +1,19 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/kauno_m_sav/kasimai,,,,,,,,,,,,,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,kasimoLokacija,,,vda_id,,,4,,,Kauno miesto savivaldybėje vykdomų kasimo darbų lokacijos,Šiame rinkinyje pateikiamos Kauno miesto savivaldybės kasimo darbų lokacijos. Lokacijos pateikiamos WGS ir LKS koordinačių sistemose. Rinkinys jungiasi su „Kasimai“ rinkinu per „prasymo_id“ raktą (kasimai.id = kasimuLokacijos.prasymo_id). 
```

Comment:
Tada dar aprašyme pataisyti turbūt reikėtų, "KasimoLokacija.prasymo_id", jeigu čia šitą rinkinį turi omenyje. 
Dar vietoj "rinkinu" turėtų būti "rinkiniu".

Reply:
Pakeista